sysutils/lima/Portfile

```
@@ -52,10 +52,12 @@ destroot {
         {*}[glob ${worksrcpath}/_output/share/doc/lima/docs/*.md] \
         ${destroot}${prefix}/share/doc/lima/docs
 
-    xinstall -m 0755 -d ${destroot}${prefix}/share/lima/examples
+    xinstall -m 0755 -d ${destroot}${prefix}/share/lima/templates
     xinstall -m 0644 \
         {*}[glob ${worksrcpath}/_output/share/doc/lima/examples/*.yaml] \
-        ${destroot}${prefix}/share/lima/examples
+        ${destroot}${prefix}/share/lima/templates
+
+    ln -fs templates ${destroot}${prefix}/share/lima/examples
```

I am unsure whether or not this is the correct way of creating a relative symlink 😅. 
It works as intended on my system, I don't suppose it would be any different on other unix systems. 
But as I am still relative new to writing Portfiles, do suggest better ways if any 😃.

---

Thanks for this change, @MidnightRocket. I'd say the link should explicitly point to the explicit final path to `templates`:

```suggestion
    ln -fs ${prefix}/share/lima/templates \
            ${destroot}${prefix}/share/lima/examples
```